exporter/exporter.go

```diff
@@ -350,77 +366,79 @@ func NewRedisExporter(redisURI string, opts Options) (*Exporter, error) {
+		"slowlog_last_id":                                    {txt: `Last id of slowlog`},
+		"slowlog_length":                                     {txt: `Total slowlog`},
+		"start_time_seconds":                                 {txt: "Start time of the Redis instance since unix epoch in seconds."},
+		"stream_group_consumer_idle_seconds":                 {txt: `Consumer idle time in seconds`, lbls: []string{"db", "stream", "group", "consumer"}},
+		"stream_group_consumer_messages_pending":             {txt: `Pending number of messages for this specific consumer`, lbls: []string{"db", "stream", "group", "consumer"}},
+		"stream_group_consumers":                             {txt: `Consumers count of stream group`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_entries_read":                          {txt: `Total number of entries read from the stream group`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_lag":                                   {txt: `The number of messages waiting to be delivered to the stream group's consumers`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_last_delivered_id":                     {txt: `The epoch timestamp (ms) of the last delivered message`, lbls: []string{"db", "stream", "group"}},
+		"stream_group_messages_pending":                      {txt: `Pending number of messages in that stream group`, lbls: []string{"db", "stream", "group"}},
+		"stream_groups":                                      {txt: `Groups count of stream`, lbls: []string{"db", "stream"}},
+		"stream_last_generated_id":                           {txt: `The epoch timestamp (ms) of the latest message on the stream`, lbls: []string{"db", "stream"}},
+		"stream_length":                                      {txt: `The number of elements of the stream`, lbls: []string{"db", "stream"}},
+		"stream_max_deleted_entry_id":                        {txt: `The epoch timestamp (ms) of last message was deleted from the stream`, lbls: []string{"db", "stream"}},
+		"stream_first_entry_id":                              {txt: `The epoch timestamp (ms) of the first message in the stream`, lbls: []string{"db", "stream"}},
+		"stream_last_entry_id":                               {txt: `The epoch timestamp (ms) of the last message in the stream`, lbls: []string{"db", "stream"}},
+		"stream_radix_tree_keys":                             {txt: `Radix tree keys count"`, lbls: []string{"db", "stream"}},
+		"stream_radix_tree_nodes":                            {txt: `Radix tree nodes count`, lbls: []string{"db", "stream"}},
+		"up":                                                 {txt: "Information about the Redis instance"},
+		"module_info":                                        {txt: "Information about loaded Redis module", lbls: []string{"name", "ver", "api", "filters", "usedby", "using"}},
+		"search_version":                                     {txt: "Information about the RediSearch module", lbls: []string{"version"}},
```

Comment:
can we bake this into the `module_info` metric? Add a `version` label? seems tightly related to the module but I don't know if they keep different versions for the two.

Reply:
It's already exists there, just in different format. Here it's `search_version:2.10.5`, but there it's `module:name=search,ver=21005,api=1,filters=0,usedby=[],using=[ReJSON],options=[handle-io-errors]`, i.e. `ver=21005`

In theory we can drop this metric and keep only module_info, but it looks nicer with dots as separators here.

Reply:
I added also now continue after registering metrics to avoid unneeded parsing of metrics after `registerConstMetricGauge`

Reply:
`In theory we can drop this metric and keep only module_info, but it looks nicer with dots as separators here.` - in that case, if the versions match, then I'd prefer to just capture one metric - the formatting is just eye candy.
But if the module version could differ from the search engine version then it'd make sense to capture both.

Reply:
> But if the module version could differ from the search engine version then it'd make sense to capture both.

Nope, that afaik shouldn't happen, removed search_version and let's keep version in module_info as is.

Reply:
👍